command/operator_debug.go

```diff
@@ -276,6 +290,14 @@ func ServerPredictor(factory ApiClientFactory) complete.Predictor {
 	})
 }
 
+// queryOpts returns a copy of the shared api.QueryOptions so
+// that api package methods can safely modify the options
+func (c *OperatorDebugCommand) queryOpts() *api.QueryOptions {
+	qo := new(api.QueryOptions)
+	*qo = *c.opts
```

Comment:
It doesn't seem to be used, but there's a `Params map[string]string` that wouldn't be copied with just this.

Reply:
Good catch. Copying that will prevent future bugs, so will fix.